ui/pages/confirmations/hooks/useCurrentConfirmation.ts

```diff
@@ -27,9 +28,15 @@ const useCurrentConfirmation = () => {
   );
   const [currentConfirmation, setCurrentConfirmation] =
     useState<Record<string, unknown>>();
+  const redesignedConfirmationsEnabled = useSelector(
+    getRedesignedConfirmationsEnabled,
+  );
 
   useEffect(() => {
-    if (!process.env.ENABLE_CONFIRMATION_REDESIGN) {
+    if (
+      !process.env.ENABLE_CONFIRMATION_REDESIGN ||
+      !redesignedConfirmationsEnabled
+    ) {
       return;
```

Comment:
[Not related to this task] Eventually we would want to refactor/simplify this file. Branching growing a lot on the useEffect.

Reply:
As we move all confirmations to new designs this branching will actually simplify.